posthog/hogql_queries/legacy_compatibility/filter_to_query.py

```diff
@@ -92,6 +93,15 @@ def legacy_entity_to_node(
 
     if entity.type == "actions":
         return ActionsNode(id=entity.id, **shared)
+    elif entity.type == "data_warehouse":
+        return DataWarehouseNode(
+            id=entity.id,
+            id_field=entity.id_field,
+            distinct_id_field=entity.distinct_id_field,
+            timestamp_field=entity.timestamp_field,
+            table_name=entity.table_name,
+            **shared,
+        )
```

Comment:
Fly-by comment: This probably needs to be handled in the reverse method `query_to_filter` and frontend side equivalents `filtersToQueryNode` / `queryNodeToFilter` as well. Potentially also in `frontend/src/scenes/notebooks/Notebook/migrations/migrate.ts` if there are faulty notebook nodes.

Reply:
Checked these and I think we're all good ✅

Reply:
Great, thanks for checking!